smurfs/src/actions/index.js

```diff
@@ -0,0 +1,43 @@
+import axios from 'axios';
+
+export const START_SMURF_FETCHING = 'START_SMURF_FETCHING';
+export const FETCH_SMURF_SUCCESS = 'FETCH_SMURF_SUCCESS';
+export const FETCH_SMURF_FAILURE = 'FETCH_SMURF_FAILURE';
```

Comment:
try to keep your naming conventions the same for your actions. aka change the 'start' to match the success & failure cases. 'FETCH_SMURF_START'